sklearn/ensemble/tests/test_iforest.py

```diff
@@ -211,3 +213,13 @@ def test_iforest_subsampled_features():
     clf = IsolationForest(max_features=0.8)
     clf.fit(X_train, y_train)
     clf.predict(X_test)
+
+
+def test_iforest_average_path_length():
+    # It tests non-regression for #8549 which used the wrong formula
+    # for average path length, strictly for the integer case
+
+    result_one = 2. * (np.log(4.) + euler_gamma) - 2. * 4. / 5.
+    result_two = 2. * (np.log(998.) + euler_gamma) - 2. * 998. / 999.
+    assert_array_almost_equal(_average_path_length(np.array([1, 5, 999])),
```

Comment:
Sorry if it was unclear, I meant to ask for the removal of the redundant `_average_path_length(np.array([1]) == _average_path_length(1)` line...

We still need to test the int arguments as they are being handled in a different line of code than if the argument is an array...

i.e. Could you add back the `assert_almost_equal(_average_path_length(1), result_one)` and `assert...(5), result_two)` lines?